pkg/api/types.go

```diff
@@ -334,6 +467,8 @@ type Capabilities struct {
 type ResourceRequirements struct {
 	// Limits describes the maximum amount of compute resources required.
 	Limits ResourceList `json:"limits,omitempty"`
+	// Requests describes the minimum amount of compute resources required.
+	Requests ResourceList `json:"requests,omitempty"`
```

Comment:
My previous comments about Request and Limit still stand, unless they were answered and I missed it.

Is this just about clarity, or is there really a semantic difference you are shooting for?